sound/pci/hda/hda_intel.c

```diff
@@ -2503,7 +2503,8 @@ static const struct pci_device_id azx_ids[] = {
 	{ PCI_DEVICE_DATA(INTEL, HDA_RPL_PX, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE) },
 	{ PCI_DEVICE_DATA(INTEL, HDA_MTL, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE) },
 	/* Lunarlake-P */
-	{ PCI_DEVICE_DATA(INTEL, HDA_LNL_P, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE) },
+	{ PCI_DEVICE_DATA(INTEL, HDA_LNL_P, AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE |
+					    AZX_DCAPS_PIO_COMMANDS) },
```

Comment:
we could also introduce a macro such as  AZX_DRIVER_LNL where all the defaults are set, that will save lines for future additions.

Reply:
Right, my optimistic thinking was that this is only going to be needed for LNL, but ACE2.0 variants will need this as well.
OK.